src/clientStream.h

```diff
@@ -56,7 +56,7 @@
  *    nodes to have any correspondence, as long as the callbacks provided are
  *    correct.
  *
- \section WhatsInANode Whats in a node
+ \section WhatsInANode What's in a node
```

Comment:
This is important. We should not break Doxygen. @mrumph, after this is addressed, please see if you can generate documentation to make sure nothing got broken (by comparing before and after snapshots of [generated] files using `diff -ur` or similar).

Reply:
The best we can do at this point is to add words (whats) to the white list.
If we want "whats" to be automatically translated to "what is", then we would need our own customized dictionary.

Reply:
Could someone point me to documentation on how to generate documentation for Squid?

Reply:
I do not know (or remember) the official answer to your question, but you can start with running `doxygen squid.dox` from the root repository directory. You may need to install "doxygen" (and "graphviz") first, but that is easy to do on Ubuntu.

There are also doxygen-related commands in doc/Programming-Guide/Makefile, but that Makefile does not work out of the box for me (because its phony targets are not marked as such?). YMMV.

Reply:
> If we want "whats" to be automatically translated to "what is", then we would need our own customized dictionary.

If that custom dictionary will _replace_ the standard one (rather than just overwrite or add a few entries), then a custom dictionary is _not_ a good solution IMO (and we should leave the desirable goal of expanding contractions alone for the time being). Otherwise, we should consider it. In other words, we are not going to maintain our own full dictionary, but we certainly can maintain a small set of the official dictionary exceptions/additions.

If a custom dictionary is not a good solution (as discussed above), then whitelisting `whats` to address this change request (assuming `What's` breaks Doxygen) is a good solution IMO.